src/python/tests/CMakeLists.txt

```diff
@@ -38,10 +38,21 @@ set(python_tests_robot
   testAmazonPod.py
 )
 
-# these tests use Atlas v5 robot model
+# Tests are run by default on the default robot
+#
+# If intending to test on a particular robot only, please set _extra_args to the
+# director argument for that robot.
+#
+# Tests can also be run on multiple robots by setting an additional
+# _extra_args_{n} to the director argument for the n-th robot to test.
+
+# these tests are tested on Atlas v5 robot model ONLY
 set(testKinematicPose_extra_args -v5)
-set(testTeleopPanel_extra_args -v5)
-set(testContinuousWalking_extra_args -v5)
+
+# these tests use both Atlas v5 and Valkyrie
+set(testTeleopPanel_extra_args_1 -v5)
+set(testContinuousWalking_extra_args_1 -v5)
```

Comment:
hmm, maybe it's confusing, but I was thinking of starting with _2.  So:

test_extra_args # applied to first test
test_extra_args_2 # applied to second test
test_extra_args_... # and so on